octoprint_mrbeam/templates/ready_to_laser.jinja2

```diff
@@ -32,12 +32,14 @@
         </div>
         <div class="modal-body">
             <img src="/plugin/mrbeam/static/img/cooling.jpg">
-            <h5 >Mr Beam need a short break to cool down. We'll continue the laser job as soon as possible. There's nothing you need to do at the moment.</h5>
+            <h5 >{{ _('Mr Beam need a short break to cool down. ') }}
```

Comment:
Why did you break it into 3 separate strings?
Also, please correct it to "Mr Beam II needs..."

Reply:
Because the line was very long, but anyway, now it is in one line